pkg/rancher-components/src/components/RcDropdown/RcDropdownMenu.vue

```diff
@@ -20,6 +20,11 @@ const hasOptions = (options: DropdownOption[]) => {
 };
 </script>
 
+<script lang="ts">
+// TODO: address the root cause of this in https://github.com/rancher/dashboard/issues/13553
+export default { inheritAttrs: false };
+</script>
+
```

Comment:
nit: We should be able to use `defineOptions` to set `inheritAttrs`[^1]:

```
defineOptions({
  inheritAttrs: false
})
```

[^1]: https://vuejs.org/guide/components/attrs#disabling-attribute-inheritance

Reply:
We should avoid `TODO`s in code.

An interesting metric is searching repos for TODOs to see how well maintained it is